src/commands/stepfunctions/instrument.ts

```diff
@@ -88,6 +90,9 @@ export class InstrumentStepFunctionsCommand extends Command {
 
     // loop over step functions passed as parameters and generate a list of requests to make to AWS for each step function
     for (const stepFunctionArn of stepFunctionArns) {
+      this.context.stdout.write(
```

Comment:
Why are we printing this now?

Reply:
For the same reason as https://github.com/DataDog/datadog-ci/pull/972/files#r1265488424